tests/custom_bootstrap.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Sonata\AdminBundle\Tests\App\AppKernel;
+
+$_ENV['KERNEL_CLASS'] = AppKernel::class;
```

Comment:
is it maybe enough to just declare those 3 in the phpunit config?

like

https://github.com/symfony/demo/blob/main/phpunit.xml.dist#L12

Reply:
Yes, that would be the best solution, but that file is handled by dev-kit

Reply:
We could add option in the dev-kit file, but I don't think it's a good idea.
It would require to put too specific code like `Sonata\AdminBundle\Tests\App\AppKernel::class` in dev-kit.

Best would be to use a `.env.test` file maybe, but didn't succeed when I tried here: https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/pull/683

Reply:
But why do you prefer here rather than in `tests/App/AppKernel.php` ? @jordisala1991

Reply:
Not sure, to me it looks uglier to override its construct (removing its parameters too). And having to add the kernel class too, it looks better having all the env vars there

Reply:
> We could add option in the dev-kit file, but I don't think it's a good idea.
> 
> It would require to put too specific code like `Sonata\AdminBundle\Tests\App\AppKernel::class` in dev-kit.
> 
> 
> 
> Best would be to use a `.env.test` file maybe, but didn't succeed when I tried here: https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/pull/683

The .env file will
only work if you install symfony dotenv and usePutEnv()

Reply:
> > We could add option in the dev-kit file, but I don't think it's a good idea.
> > It would require to put too specific code like `Sonata\AdminBundle\Tests\App\AppKernel::class` in dev-kit.
> > Best would be to use a `.env.test` file maybe, but didn't succeed when I tried here: [sonata-project/SonataDoctrinePhpcrAdminBundle#683](https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/pull/683)
> 
> The .env file will
> only work if you install symfony dotenv and usePutEnv()

Would it be better to do it this way ?

The code could be added to dev kit
```
if (file_exist(.env.test)) {
    ...
}
```